back/engines/commercial/multi_tenancy/app/services/multi_tenancy/templates/serializer.rb

```diff
@@ -690,22 +672,22 @@ def yml_idea_statuses
       end
 
       def yml_ideas
-        Idea.published.map do |i|
+        Idea.published.map do |idea|
           yml_idea = {
```

Comment:
This mapping seems to be shared between project_copy_service and serializer, could this not be moved in the template service?

Reply:
We could potentially move a lot of code to the shared template service, as there is a lot of code duplication between both. I think this might be too much out of scope work for this ticket, and by allowing some degree of code duplication, the structure of the template remains more explicit and readable.

Reply:
OK sure. Let's leave it as it is.